Sources/Apodini/Configurations/Configuration.swift

```diff
@@ -71,14 +71,10 @@ public struct EmptyCommand: ParsableCommand {
 
 extension Array: Configuration where Element == Configuration {
     public func configure(_ app: Application) {
-        forEach {
-            $0.configure(app)
-        }
+        forEach { $0.configure(app) }
```

Comment:
SwiftLint didn't complain so I'd assumed it'd be fine. (tbh if this were up to me we'd be using classic `for` loops instead of `forEach` where ever possible, but I'm not gonna make that change since it's not at all up to me to make that decision)